File: Data Structures & Algorithms/max-consecutive-ones-iii/submission-0.java
class Solution {
    public int longestOnes(int[] nums, int k) {
        int n = nums.length;
        int count = 0;
        int left = 0;
        int zero = 0;
        for(int right = 0;right<n;right++)
        {
            if(nums[right]==0) zero++;
            while(zero>k)
            {
                if(nums[left]==0)zero--;
                left++;
            }
            count = Math.max(count, right-left+1);
        }
        return count;
    }
}

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int left=0;
        int right=1;
    vector<int>unique;
    unique.push_back(nums[left]);
        while(right<nums.size())
        {
            if(nums[left]!=nums[right])
            {
                unique.push_back(nums[right]);
                left=right;
            }
            right++;
            
        }
        nums=unique;
        return unique.size();
    }
};